client/web/src/search/input/LoggedOutHomepage.tsx

```diff
@@ -35,14 +36,88 @@ const exampleQueries = [
     },
 ]
 
-export interface LoggedOutHomepageProps extends TelemetryProps, ThemeProps {}
+const exampleNotebooks = [
```

Comment:
nit: add typing for clarity? (and same for `exampleQueries` above?)

```suggestion
const exampleNotebooks: SearchExample[] = [
```